CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+14.4.1
```

Comment:
@lstp Once we solve the issue, are we planning to keep the extra logging or remove it?

(This is a public gem)

Reply:
I think it's useful to keep them but no objections to remove them if required.